### ISSUE TITLE
Should I train again on my dataset？

### ISSUE THREAD

**Issue by EmilyLike** [ChenyangLEI/Fully-Automatic-Video-Colorization-with-Self-Regularization-and-Diversity#23]
Hello， your work is so interesting. I wanna know, if I wanna use your mehtod to compare with my method. But my training dataset and test data are different from yours, should I use my dataset to train your model again?

**Comment by ChenyangLEI** [comment]
Of course. Please make sure that the training set, validation set and test set are consistent among baselines.
